doc/dev_guide/internals/box_protocol.rst

```diff
@@ -1102,8 +1101,8 @@ If instead we said |br|
 :code:`conn:prepare([[SELECT dd, дд AS д FROM t1;]])` |br|
 then we could get almost the same response, but there would
 be no IPROTO_DATA and there would be two additional items: |br|
-``34 00 = IPROTO_BIND_COUNT + MP_UINT = 0`` (there are no parameters to bind), |br|
-``33 90 = IPROTO_BIND_METADATA + MP_ARRAY, size 0`` (there are no parameters to bind).
+``34 00 = IPROTO_BIND_COUNT and MP_UINT = 0`` (there are no parameters to bind), |br|
+``33 90 = IPROTO_BIND_METADATA and MP_ARRAY, size 0`` (there are no parameters to bind).
```

Comment:
Do you mean that byte 34 is IPROTO_BIND_COUNT and byte 00 is MP_UINT? Which one of them equals zero — only MP_UINT or both?

It is not clear how the equals sign works in this case, what parts are code and what are not. Same comment for strings 1663–64.

Our documentation guidelines provide some [information on how to write about code](https://www.tarantool.io/en/doc/latest/contributing/docs/markup/code/), what is code and what is not.

Reply:
At the start we say IPROTO_BIND_COUNT is a value i.e. 34
and MP_UINT means unsigned integer i.e. MP_UINT is a type.
At the end we have an illustration:
    34                         IPROTO_BIND_COUNT
    00                         MP_INT = 0 = number of parameters to bind
    33                         IPROTO_BIND_METADATA
    90                         MP_ARRAY, size 0 = there are no parameters to bind
Incidentally on 2020-08-16 I wrote to E. Shebunyaeva suggesting I could
change occurrences of MP_INT to MP_UINT, but there was no decision.